version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.11.0"
+version in ThisBuild := "0.11.0 -SNAPSHOT"
```

Comment:
Yeah, I keep getting a "Unresolved dependencies: algebird-test" error when I don't use "-SNAPSHOT". Is this happening to other people too?